src/components/ContentUploader/ContentUploader.js

```diff
@@ -76,7 +76,7 @@ type Props = {
     uploadHost: string,
     useUploadsManager?: boolean,
     dataTransferItems: Array<DataTransferItem | UploadDataTransferItemWithAPIOptions>,
-    isDraggingItemsToUploadsManager?: boolean
+    isDraggingItemsToUploadsManager: boolean
```

Comment:
I'm fixing the related flow issue in https://github.com/box/box-ui-elements/pull/465, you can revert this line